<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserChange</class>
 <widget class="QMainWindow" name="UserChange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="lbl_login">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>23</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>UserName - Qwerty</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" rowspan="2">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Blocking</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QPushButton" name="btn_unlock">
           <property name="text">
            <string>Unblock</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="btn_lock">
           <property name="text">
            <string>Block</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="btn_time_block">
           <property name="text">
            <string>Block untill</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDateTimeEdit" name="dateTimeEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Change Type</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QComboBox" name="comboBox_userType">
           <item>
            <property name="text">
             <string>Client</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Provider</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Admin</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_apply">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btn_delete">
        <property name="text">
         <string>Delete User</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
